.github/workflows/github-test.yaml

```diff
@@ -16,12 +16,15 @@ permissions:
 
 defaults:
   run:
-    shell: bash
```

Comment:
I think this was always superfluous, but especially if we are testing on windows